pkgs/top-level/release-haskell.nix

```diff
@@ -520,7 +520,26 @@ let
               ghc948
               ;
           };
-        };
+        }
+        //
+          removePlatforms
+            [
+              # Testing cross from x86_64-linux
+              "aarch64-darwin"
+              "aarch64-linux"
+              "x86_64-darwin"
+            ]
+            {
+              haskell.packages.ghc912 = {
+                inherit (packagePlatforms pkgs.pkgsCross.aarch64-multiplatform.haskell.packages.ghc912)
+                  ghc
+                  hello
+                  microlens
+                  miso
+                  reflex-dom
```

Comment:
`reflex-dom` currently doesn't eval because libsoup is marked as insecure. We should probably remove it from here.

Reply:
I removed `reflex-dom` here and also used the default `haskellPackages` for the release-haskell file cross to aarch64 thing instead of pinning 9.12.